controllers/module_reconciler.go

```diff
@@ -265,6 +265,8 @@ func (r *ModuleReconciler) handleBuild(ctx context.Context,
 		r.metricsAPI.SetCompletedStage(mod.Name, mod.Namespace, kernelVersion, metrics.BuildStage, false)
 	case build.StatusCompleted:
 		r.metricsAPI.SetCompletedStage(mod.Name, mod.Namespace, kernelVersion, metrics.BuildStage, true)
+	case build.StatusFailed:
+		logger.Info(utils.WarnString("Build job has failed. If the fix is not in Module CR, then delete job after the fix in order to restart the job"))
```

Comment:
Should we return an error here? Otherwise I believe we'll proceed with the DaemonSet creation like if the image was successfully built.

Reply:
@ybettan , Quentin's comment was based on the previous version of the PR. In the new version, we either return if Build/Sign has completed successfully or not, and based on that proceed to DaemonSet or not.

Reply:
I am not sure it was. I can still see this code and there is no `outdated` badge on this comment.

Reply:
but the code surrounding this function has changed and the meaning of the boolean returned by this function has changed